common/src/main/scala/explore/contexts.scala

```diff
@@ -4,16 +4,32 @@
 package explore
 
 import cats.effect.IO
+import cats.syntax.all._
 import crystal.react.Ctx
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string._
+import explore.model.Help
 import japgolly.scalajs.react.Reusability
+import japgolly.scalajs.react.Reusability._
 import monocle.macros.Lenses
+import sttp.model.Uri
 
 object AppCtx extends Ctx[IO, AppContextIO]
 
 @Lenses
-case class HelpContext(msg: Option[String])
+case class HelpContext(
+  rawUrl:        Uri,
+  editUrl:       Uri,
+  user:          String Refined MatchesRegex["[\\w-_]*"],
```

Comment:
If the regexes are `+` instead of `*` we can prevent empty strings.

Reply:
You're right. good idea